### DISCUSSION TITLE
Action stuck for 4 days with Local Runner (Windows)

### DISCUSSION THREAD

**Discussion by bfloch** [community/community#27171] [Actions]
<p>In one of our private repos the action is stuck and does not finish for 4 days!!!</p>
<p>No way to cancel.</p>
<p>Other queued jobs are not running. The local runner - even after restart - claims: “Listening for Jobs”.</p>
<p>What to do?</p>

**Comment by bfloch** [comment]
<p>Tried disabling Actions for the repo. Re-enabling leaves the same state.</p>
<p>The stuck actions is essentially done (failed) and hangs in this step:</p>
<p>Complete Job</p>
<p>   “Cleaning up orphan processes”</p>

**Comment by lokesh755** [comment]
<p>Could you provide the URL of the run which got stuck ?</p>

**Comment by bfloch** [comment]
<p>Thanks for the quick response:</p>
<p><a href="https://github.com/bfloch/blender2.8/commit/0d2953fb40e147d0ab27d29ed38fdd15ddce766b/checks?check_suite_id=301138635" rel="nofollow noopener">https://github.com/bfloch/blender2.8/commit/0d2953fb40e147d0ab27d29ed38fdd15ddce766b/checks?check_suite_id=301138635</a></p>

**Comment by bfloch** [comment]
<p>I think I can replicate the issue.</p>
<p>It happened here again (public repo this time):</p>
<p><a href="https://github.com/bfloch/blender/commit/30de5c3ac775f98f2751b2bc74bbad2ba3803486/checks?check_suite_id=307625362" rel="nofollow noopener">https://github.com/bfloch/blender/commit/30de5c3ac775f98f2751b2bc74bbad2ba3803486/checks?check_suite_id=307625362</a></p>
<p>I fear it is windows related as I was in the _work folder and as such the runner probably could not clean up due to Microsoft’s famous file locking.</p>
<p>After I left the subfolder the runner proceeded, however the job is in limbo now.</p>
<p>Cancelling or even disconnecting the runner will not clear the status - and as such it will block everything else.</p>
<p>So basically we have a guaranteed way to limbo github actions.</p>

**Comment by changhuixu** [comment]
<p>Same here.</p>
<p><a href="https://github.com/changhuixu/DelimitedDataFileHelper/actions" rel="nofollow noopener">https://github.com/changhuixu/DelimitedDataFileHelper/actions</a></p>
<p>If it is possible, could you help me remove all “Queued” and “stuck” actions?</p>
<p>Thank you.</p>

**Comment by bfloch** [comment]
<p>Not sure if this was someones doing or not but the public repo and the other commenters repo seem to be solved.</p>
<p>My private repo is still in limbo since 6 days:</p>
<p><a href="https://github.com/bfloch/blender2.8/actions" rel="nofollow noopener">https://github.com/bfloch/blender2.8/actions</a></p>
<p>Basically I am blocked since two days and can not proceed with my work.</p>
<p>Any help?</p>

**Comment by bfloch** [comment, marked as answer]
<p>For reference:</p>
<p>github support told me that while certain workflows still look like they would stall, you can create a new run and it would work. Even, like in my case when there seem to be queued jobs.</p>
<p>Just trigger a new job and it will run instead. This worked for me.</p>

**Comment by ericwhitefield** [comment]
<p>I have seen the same issue with a self-hosted MacOS.</p>
<p>Similar to the solution below. I was able to run a second workflow, even though the statuses incorrectly showed ‘running’ and ‘queued’. By the next day the statuses finally updated.</p>
<p>This was only the status in the action list on GitHub that was incorrect. Viewing the details revealed that all steps had finished.</p>

**Comment by vleandersson** [comment]
<p>I don’t think this should be counted as a solution, but rather a workaround for the actual issue.</p>

**Comment by unfor19** [comment]
<p>I encounter this issue usually with Windows runners, not sure if it’s still relevant, here are the details:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/unfor19/install-aws-cli-action-test/runs/1602486495?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/15122452/181102154-dd097412-2a1e-406e-90fb-5411437d1696.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/unfor19/install-aws-cli-action-test/runs/1602486495?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p><a href="https://github.com/unfor19/install-aws-cli-action-test/runs/1602486495?check_suite_focus=true" target="_blank" rel="noopener nofollow ugc">19/install-aws-cli-action-test/runs/1602486495</a></p>

  <p><span class="label1">GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/9/e990a4bb1e6783d9146420ba18ac9fb78646b547.png" data-download-href="/uploads/short-url/xkd9aiPdy9J4FFfTA5gpYR9jnRZ.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15122452/181102155-195169ee-f38b-4b9f-a1b1-a8f0ed193a27.png" alt="image" data-base62-sha1="xkd9aiPdy9J4FFfTA5gpYR9jnRZ" width="690" height="277" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/9/e990a4bb1e6783d9146420ba18ac9fb78646b547_2_690x277.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/9/e990a4bb1e6783d9146420ba18ac9fb78646b547_2_1035x415.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/9/e990a4bb1e6783d9146420ba18ac9fb78646b547_2_1380x554.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/9/e990a4bb1e6783d9146420ba18ac9fb78646b547_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2484×1000 199 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by unfor19** [comment]
<p>I found out my issue - I was running <code>msiexec</code> and apparently it causes issues, probably because it attempts to open a new window while it’s running on a headless machine.</p>
